### PR TITLE
Add Write File

### PR DESCRIPTION
**Pull request by willmichael**
Adds write file feature to the plugin

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.example.plugin_tozny">
-    <uses-sdk android:minSdkVersion="23"
+    <uses-sdk android:minSdkVersion="26"
```
Comment (efabens) [inline android/src/main/AndroidManifest.xml]: Do you know what causes this bump?

**Review by efabens — COMMENTED** [review]
Code looks fine. I am curious what caused the need to bump to sdk version 26? I think that maybe higher than we want to currently go if we can avoid it.

**Review by galxy25 — APPROVED** [review]

File: android/src/main/AndroidManifest.xml (hunk 1 above)
Comment (willmichael) [reply android/src/main/AndroidManifest.xml]: Yeah so its the snippet that allows easy serializing:
```    
private static DateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZZZZ");
    @RequiresApi(api = Build.VERSION_CODES.O)
    static String recordMetaToJson(RecordMeta meta) {
        Map<String, Object> metaMap = new HashMap<>();
        metaMap.put("record_id", meta.recordId().toString());
        metaMap.put("user_id", meta.userId().toString());
        metaMap.put("writer_id", meta.writerId().toString());
        metaMap.put("created", dateFormatter.format(meta.created()));
        metaMap.put("last_modified", dateFormatter.format(meta.lastModified()));
```

Comment (willmichael) [reply android/src/main/AndroidManifest.xml]: Build.VERSION_CODES.O -> Version Oreo which is android 8 and up. With android currently sitting on version 11 I assumed its relatively safe for users, but we don't have a clear directive on what version we're supporting so good callout

**Review by lkwdwrd — APPROVED** [review]
As long as the version hits our requirements, looks good.

**Review by rosasck — APPROVED** [review]

**Pull request merged**